recordedit/form.controller.js

```diff
@@ -214,9 +213,8 @@
                 vm.resultset = true;
                 // delay updating the height of DOM elements so the current digest cycle can complete and "show" the resultset view
                 $timeout(function() {
-                    mainBodyEl = $document[0].getElementsByClassName('main-body')[1];
-                    setMainContainerHeight();
-                    UiUtils.setFooterStyle(1);
+                    detachResizeSensors();
+                    setResizeSensors();
```

Comment:
You're right. technically we don't need to detach the resize events but since in this case, the DOM is completely changing I felt like maybe I should detach them.